README.md

```diff
@@ -403,8 +403,8 @@ const payWithTransfer = await fincra.collection.payWithTransfer(data);
 | **Parameters**      | **Data type** | **Required** | **Description**                                           |
 | ------------------- | ------------- | ------------ | --------------------------------------------------------- |
 | `expiresAt`         | string        | true         | The expiry of the virtual account in minutes.             |
-| `name`              | string        | false        | The name that should be on the account.                   |
-| `merchantReference` | string        | false        | The unique identifier of the transaction on your system . |
```

Comment:
Hello @softmuneeb 

Thanks for the catch!

it would seem that there have been mutliple updates since this method was last updated.

Do you think you can dive into the code to update this or you are just okay with updating the **README.md**(might have to find a weekend to make the update if otherwise)